controllers/helper.js

```diff
@@ -111,6 +111,8 @@ const handleFileUpload = function (req, res, next) {
     if (req.query.encryptfiles) {
       return periodic.core.files.uploadMiddlewareHandler({
         periodic,
+        sizeLimit: 10000000, // 10mb in bytes

```

Comment:
make this and acceptedMIMETypes db configurable, you have access to extsettings on line 97